sopel/loader.py

```diff
@@ -104,10 +104,35 @@ def clean_callable(func, config):
 
 
 def is_triggerable(obj):
-    return any(
-        hasattr(obj, attr)
-        for attr in ('rule', 'event', 'intents', 'commands',
-                     'nickname_commands'))
+    """Check if ``obj`` can handle the bot's triggers.
+
+    :param obj: any :term:`function` to check.
+    :return: ``True`` if ``obj`` can handle the bot's triggers.
+
+    A triggerable is a callable that will be used by the bot to handle a
+    particular trigger (i.e. an IRC message): it can be a regex rule, an event,
+    an intent, a command or nickname command. However, it must not be a job
+    or an URL callback.
+
+    .. seealso::
+
+        The :mod:`sopel.module` defines decorators to make a function a
```

Comment:
```suggestion
        Many of the decorators defined in :mod:`sopel.module` make the decorated function a
```
(+ wrap)

Ultimately this is just to get away from saying "The sopel.module defines", because that shouldn't start with "The", but I got carried away making the whole sentence sound nicer. 😸